src/Elasticsearch.Net/Responses/DynamicResponse.cs

```diff
@@ -295,18 +273,14 @@ public bool Remove(KeyValuePair<string, dynamic> item)
 		/// Gets an <see cref="T:System.Collections.Generic.ICollection`1"/> containing the values in the <see cref="DynamicResponse"/>.
 		/// </summary>
 		/// <returns>An <see cref="T:System.Collections.Generic.ICollection`1"/> containing the values in the <see cref="DynamicResponse"/>.</returns>
-		public ICollection<dynamic> Values
-		{
-			get { return this.BackingDictionary.Values; }
-		}
+		public ICollection<dynamic> Values => this.BackingDictionary.Values;
 
 		private static KeyValuePair<string, dynamic> GetDynamicKeyValuePair(KeyValuePair<string, dynamic> item)
 		{
-			var dynamicValueKeyValuePair =
-				new KeyValuePair<string, dynamic>(item.Key, new ElasticsearchDynamicValue(item.Value));
-			return dynamicValueKeyValuePair;
+			return new KeyValuePair<string, dynamic>(item.Key, new ElasticsearchDynamicValue(item.Value));
 		}
 
+		//TODO:For what purpose do you use this function?

```

Comment:
Our DynamicDictionary is a source copy of NancyFX's https://github.com/NancyFx/Nancy/wiki/Taking-a-look-at-the-DynamicDictionary, this method might have made sense at some point.